src/misc/cache/lv_image_cache.h

```diff
@@ -13,7 +13,11 @@ extern "C" {
 /*********************
  *      INCLUDES
  *********************/
-#include "lv_cache_private.h"
+
+#include "../../lv_conf_internal.h"
+#include <stdbool.h>
```

Comment:
Need to remove it and include lv_types.h where lv_result_t is defined.

Reply:
lv_types.h doesn't supply `bool`

Reply:
Emmm. https://github.com/lvgl/lvgl/pull/5767 is still not merged.

Reply:
I think if this pr is not merge, we can update this fix later.